src/docfx/lib/json/YamlUtility.cs

```
@@ -68,41 +68,34 @@ public static T DeserializeData<T>(string data, FilePath? file) where T : class,
         /// </summary>
         public static T DeserializeData<T>(TextReader data, FilePath? file) where T : class, new()
         {
-            var (_, token) = ParseCore(data, file);
-            return token?.ToObject<T>(JsonUtility.Serializer) ?? new T();
+            return ParseCore(new ErrorList(), data, file)?.ToObject<T>(JsonUtility.Serializer) ?? new T();
```

`ErrorList` is used to ignore the errors?